docs/integrations/fetching-incidents.md

```diff
@@ -124,6 +124,69 @@ If you do not have any incidents to return then just return an empty list to ```
 demisto.incidents([])
 ```
 
+## Advenced
+
+### Generic fetch with look back
+Use the generic look-back methods for fetching missing incidents.
+
+#### Use case
+During a **fetch-incidents** run, some edge-cases might cause missing incidents from the 3rd-party product. The most common are:
+* Indexing issues in the product: for example, if an incident A was created before an incident B, and only B was indexed, the fetch-incidents will fetch only B and will miss A, so if A was indexed after the fetch was called, the next fetch will fetch only from the created time of B.
+* An update in the incident information: some implementations of fetch-incidents are using a query filter to fetch only specific incidents. If initially an incident did not match the query (i.e., it was not fetched) but at some point was updated so that now it does match the query, a **fetch-incidents** command that queries incidents from the product using a **time** field, will not find it.
+
+#### Solution
+Having the *look_back* parameter, we can configure how much the fetch-incidents will look back in time (minutes) to get the incidents that were created a while ago but indexed a minute ago.
+
+The generic methods for this feature are implemented in [CommonServerPython](https://xsoar.pan.dev/docs/reference/api/common-server-python) and you can find the exact documentation there, but here is a short description.
+
+* **get_fetch_run_time_range()** - using the last run object and other necessary parameters, this method calculates and retrieves the time range from which to fetch.
+If the parameter look_back is given, then the start time will always be >= than `now - look_back`.
+
+* **filter_incidents_by_duplicates_and_limit()** - After getting the incidents from the 3rd party API call, we need to filter out the duplicate incidents.
+In the example above, after an incident A is indexed the fetch will bring incidents A and B, then we must return only A because B was already fetched. In addition, after filtering duplicates, if we still have more incidents than the limit, the function will return only by the limit.
```

Comment:
```suggestion
In the example below, after an incident A is indexed the fetch will bring incidents A and B, then we must return only A because B was already fetched. In addition, after filtering duplicates, if we still have more incidents than the limit, the function will return only by the limit.
```

don't you mean the example below (python code)?

Reply:
I meant the example in line 134.